src/live.html

```diff
@@ -179,18 +188,20 @@ <h1 id="title">Live</h1>
 						<img src="assets/img/resrborder.png"/>
 					</div>
 				</li>
-
 				<li data-bind-view="hardware">
-					<a href="#/hardware">Hardware</a>
+					<a href="#/hardware">Hardware List</a>
 				</li>
 				<li data-bind-view="challenges">
-					<a href="#/challenges">Challenges</a>
+					<a href="#/challenges">Challenges & Prizes</a>
```

Comment:
On mobile it actually looks fine but we should go for one option only

Reply:
Changed to Prizes only (as it's easier to understand imo)

Reply:
hmm but isn't the challenges what you want to show? I think that challenges > prizes

Reply:
+1 to challenges

Reply:
sigh, we don't refer to challenges as they are more like prizes sponsors give out. I believe Challenges can be a bit weirder to explain. We have always called them prizes I believe. But sure we can change it back.